packages/apps/explorer/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@kadena/block-explorer",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prisma:generate": "npx prisma generate",
+    "create:info:test": "curl --request GET 'http://web:3000/api/info/create?network=Testnet'",
+    "create:info:main": "curl --request GET 'http://web:3000/api/info/create?network=Mainnet'"
+  },
+  "dependencies": {
+    "@prisma/client": "^3.13.0",
+    "@trendyol-js/react-carousel": "^2.0.1",
+    "async": "^3.2.3",
+    "axios": "^0.25.0",
```

Comment:
Discussion: would it be nice to use fetch (i.e. cross-fetch) everywhere?

Reply:
Yes agreed but this was for a quick merge as is so we can setup the whole deployment structure.
We do indeed need cleaning afterwards